Simplified/NYPLHoldsViewController.m

```diff
@@ -106,7 +106,11 @@ - (void)viewDidLoad
                        action:@selector(didSelectSearch)];
   self.searchButton.accessibilityLabel = NSLocalizedString(@"Search", nil);
   self.navigationItem.rightBarButtonItem = self.searchButton;
-  
+}
+
+- (void)viewWillAppear:(BOOL)animated
+{
+  [super viewWillAppear:animated];
   if([NYPLBookRegistry sharedRegistry].syncing == NO) {
     [self.refreshControl endRefreshing];
     [[NSNotificationCenter defaultCenter] postNotificationName:NYPLSyncEndedNotification object:nil];
```

Comment:
Just missed adding the line to update the collection view content offset:
`self.collectionView.contentOffset = CGPointZero;`